js/main.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
+function getNumber (min, max) {
+  const randomNumber = Math.random();
+
+  if (min > max  ) {
+    console.log('Ошибка');
```

Comment:
в целом мы можем вывести консоль, но я бы вернул 0
(и тот, и тот вариант валидный, но лучше когда функция всегда что-то нормальное возвращает)
```suggestion
    return 0
```